test/Microsoft.AspNetCore.Sockets.Client.Tests/LongPollingTransportTests.cs

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Channels;
+using Microsoft.AspNetCore.Sockets.Internal;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Moq.Protected;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Sockets.Client.Tests
+{
+    public class LongPollingTransportTests
+    {
+        [Fact]
+        public async Task LongPollingTransportStopsPollAndSendLoopsWhenTransportDisposed()
+        {
+            var mockHttpHandler = new Mock<HttpMessageHandler>();
```

Comment:
This seems to be repeated. Can we move some of this set up into a helper method?